### DISCUSSION TITLE
Чек-лист для выходной двери

### DISCUSSION THREAD

**Discussion by Assargadon** [hackerembassy/space#3] [Ideas]
Достаточно стандартный приём, насколько я могу судить.
Последний уходящий пробегается по чек-листу, чтобы не забыть чего-нибудь.

Важно, я полагаю, не перегружать список - это было бы нечестно по отношению к последнему. Например, полагаю, _"помыть посуду"_ - чересчур, а _"сложить посуду со столов в раковину"_ - нормально.

**Comment by tectonick** [comment]
- Выключить свет (не забыть второй этаж)
- Убедится, что двери на втором этаже тоже закрыты
- Убедиться, что паяльное оборудование выключено
- Опустить темературу воды в бойлере до 38.
- Проверить, чтобы не тек унитаз
- Выключить 3d принтер снизу
- Убедиться, что контроллер замка online.
- Выгнать всех животных
- Если много мусора в пакете - вынести
- Сложить посуду со столов в раковину
- Выключить кондиционер на втором этаже
